client/stylesheets/utils/_fonts.import.less

```diff
@@ -1,72 +1,47 @@
 @font-face {
```

Comment:
This file contained Roboto font declarations, but was never referenced so I assumed the contents could be replaced.  I replaced the values using webfont-dl generated css.